src/main/java/org/isf/accounting/gui/PatientBillEdit.java

```diff
@@ -112,6 +112,8 @@
 public class PatientBillEdit extends JDialog implements SelectionListener {
 
 	private static final Logger LOGGER = LoggerFactory.getLogger(PatientBillEdit.class);
+	private static final ImageIcon admissionIcon = new ImageIcon("rsc/icons/bed_icon.png");
+	private static final String textOPD = MessageBundle.getMessage("angal.common.opd.txt");
```

Comment:
The variable names do not follow the standards for `static final` variables; see `LOGGER`